futurenhs.app/components/_pageTemplates/GroupHomeTemplate/index.tsx

```diff
@@ -110,7 +161,7 @@ export const GroupHomeTemplate: (props: Props) => JSX.Element = ({
                                 pageId: contentPageId,
                             }).then((response) => {
                                 const updatedBlocks: Array<CmsContentBlock> =
-                                    response.data
+                                    response.data.content.blocks
```

Comment:
perhaps some optional chaining here?

Reply:
I'd rather it errored because these should be present.

A thing to be careful of with optional chaining (which has bitten me in the past) - it's tempting to use it to avoid errors occurring, but used in the wrong situations it will cause the app to silently not work correctly without errors being thrown to aid debugging

Reply:
Ah good to know! I definitely need to keep this in mind, I may be over using it now you mention that.